ohsome_quality_api/geodatabase/client.py

```diff
@@ -69,50 +69,74 @@ async def get_shdi(bpoly: Feature | FeatureCollection) -> list[Record]:
         return await conn.fetch(query, geom)
 
 
-async def get_building_area(bpoly: Feature) -> list[Record]:
+async def get_building_area(bpoly: Feature, table_names: list[str]) -> list[Record]:
     """Get area of building footprints for a bounding polygon."""
+    area_list = []
```

Comment:
dict instead of list? then its easier to differentiate.

Reply:
remember to change return type

Reply:
Is this function used anywhere else than in the test 'test_get_building_area' from 'test_geodatabase'? Should the test rather use function 'get_reference_building_area' from indicator or should the indicator use the function from the client, but both doesn't make sense?